test/test_jit.py

```diff
@@ -1469,7 +1469,7 @@ def __init__(self):
             def forward(self, x):
                 x = x.quantize_linear(1.0, 0, torch.uint8)
                 x = x.int_repr()
-                x = x.dequantize_linear(1.0, 0, torch.uint8)
+                x = x._dequantize_linear(1.0, 0, torch.uint8)
```

Comment:
why do we still have this? should be calling torch._dequantize_linear(...)

Reply:
one more here

Reply:
yeah once this is changed, we can remove it from tensor methods also right